Automattic/jetpack
# SSO: fix settings toggles inconsistency

SSO settings ("Match by email" and "Require Two-Step Authentication") are preserved if you turn off the module.
However, they always appear disabled if you turn them back on, you have to reload the page to see the stored values.

https://user-images.githubusercontent.com/1341249/202568227-f7846984-1fd1-452c-a8cf-bcb1051150cd.mov

#### Changes proposed in this Pull Request:
* Make the SSO settings always reflect the actual value if the module is enabled.

#### Other information:


#### Jetpack product discussion
Discussion with the design team on the best UX: p1668719785659149-slack-C0D96691V

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
1. Go to "Jetpack -> Settings -> Security -> WordPress.com login".
2. Try turning on and off the module and its settings, occasionally reloading the page.
3. Make sure that the settings are always off when the module is disabled, and reflect the actual values when the module is on, whether you reload the page or not.
